### PR TITLE
Scrypt docs code example contradict RFC 7914 (#3302)

### PR DESCRIPTION
**Pull request by Badg**

### CODE REVIEW

**Comment by alex** [comment]
Jenkins, ok to test.

**Comment by Badg** [comment]
Hold off on merging this; I want to add a saner value for n as well (1024 is insufficient).

File: docs/hazmat/primitives/key-derivation-functions.rst [hunk 1]
```diff
@@ -789,14 +788,16 @@ Different KDFs are suitable for different tasks such as:
...
+    the 3 parameters: ``n``, ``r``, and ``p``. In general, the memory cost of
+    Scrypt is affected by the values of both ``n`` and ``r``, while ``n`` also
+    determines the number of iterations performed. ``p`` increases the
+    computational cost without affecting memory usage. A more in-depth
+    explanation of the 3 parameters can be found `here`_.
 
-    :rfc:`7914` `recommends`_ values of r=8 and p=1 while scaling n to the
-    number appropriate for your system.
+    :rfc:`7914` `recommends`_ values of ``r=8`` and ``p=1`` while scaling ``n``
+    to a number appropriate for your system. `Scrypt literature`_ suggests a
```
Comment (reaperhulk) [inline docs/hazmat/primitives/key-derivation-functions.rst]: Let's change "Scrypt literature" to "The scrypt paper"

**Comment by Badg** [comment]
**Of course** when I skip building the docs locally I forget to update the reference text. Womp womp.

**Comment by Badg** [comment]
I'm having a hard time deciphering the failure; can anyone help? It's in the [py2.7 docs build](https://travis-ci.org/pyca/cryptography/jobs/182924235). I can't run doctests locally because my openSSL is too old for scrypt. But the log makes it seem like doctests passed?

**Comment by reaperhulk** [comment]
It doesn't think `logins` is a word. Go ahead and add it to the spelling_wordlist.txt

**Pull request merged**